ddl/ddl_api.go

```diff
@@ -1455,7 +1456,22 @@ func modifiable(origin *types.FieldType, to *types.FieldType) error {
 			return nil
 		}
 	case mysql.TypeEnum:
-		return errUnsupportedModifyColumn.GenWithStackByArgs("modify enum column is not supported")
+		if origin.Tp == to.Tp {
+			if len(to.Elems) < len(origin.Elems) {
+				msg := fmt.Sprintf("elem length %d is less than origin %d", len(to.Elems), len(origin.Elems))
```

Comment:
How about:
msg := fmt.Sprintf("enum column elements length %d is less than origin %d", len(to.Elems), len(origin.Elems))